.buildkite/pipeline.yml

```diff
@@ -84,10 +93,21 @@ steps:
 
       - label: "Richards Runoff CPU"
         command: "julia --color=yes --project=.buildkite experiments/standalone/Soil/richards_runoff.jl"
-        artifact_paths: "experiments/standalone/Soil/artifacts/*png"
+        artifact_paths: "experiments/standalone/Soil/artifacts/cpu/*png"
```

Comment:
I changed the artifact paths to use the architecture type. Even though we aren't producing artifacts on GPU currently, I think this is a useful separation